### PR TITLE
Require trunclenf and trunclenr to be set both or none

### LINKED ISSUE #454: --trunclenf or --trunclenr is silently ignored when just one is given
### Description of the bug

--trunclenf or --trunclenr is silently ignored when just one is given

### Command used and terminal output

```console
#ignores --trunclenf / --trunclenr:
nextflow run nf-core/ampliseq -r 2.3.2 -profile test,singularity --outdir results --trunclenf 200
nextflow run nf-core/ampliseq -r 2.3.2 -profile test,singularity --outdir results --trunclenr 200

#works:
nextflow run nf-core/ampliseq -r 2.3.2 -profile test,singularity --outdir results --trunclenf 200 --trunclenr 200
```

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by d4straub**
Closes #454. 
An error will occur when only one of `--trunclenf` and `--trunclenr` is set. Before, if only one of those were set, the value would be silently ignored.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by erikrikarddaniel — COMMENTED** [review]
Shouldn't `--single_end` be taken into account, so that when that is set, only `--trunclenf` is required?

**Comment by d4straub** [comment]
Thats right, will adjust this in the next days!

**Comment by d4straub** [comment]
Alright, now `--trunclenf` & `--trunclenr` are not required when having single ended data. Additionally, there is a warning that `--trunclenr` is ignored when set and single end data is used.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#448](https://github.com/nf-core/ampliseq/pull/448) - Updated SBDI export scripts to include Unite species hypothesis information if available.
 - [#451](https://github.com/nf-core/ampliseq/pull/451) - Pairwise statistics will be now performed on a subset of metadata columns specified with `--metadata_category` instead of ignoring that setting.
 - [#451](https://github.com/nf-core/ampliseq/pull/451) - Replace busybox with Ubuntu base image for GCP support.
+- [#455](https://github.com/nf-core/ampliseq/pull/455) - Stop with descriptive error when only one of `--trunclenf` and `--trunclenr` is given, before it was silently ignored.
```
[the remark this answers is not in the document]
Comment (d4straub) [reply CHANGELOG.md]: Thanks, done.

**Pull request merged**